pyhive/tests/test_hive.py

```diff
@@ -18,7 +18,8 @@
 _HOST = 'localhost'
 
 
-@unittest.skipIf(sys.version_info.major == 3, 'Hive not yet supported on Python 3')
+@unittest.skipIf(sys.version_info.major == 3,
```

Comment:
Agreed... but the thrift-sasl package does not seem to get a whole lot of attention, and so far my upstream patch has yet to be accepted. I'll see if I can hack the setup.py to install my branch on this project, so we can at least see if it works.